mscore/inspector/inspector_chord.ui

```diff
@@ -49,7 +49,6 @@
      </property>
      <property name="font">
       <font>
-       <weight>75</weight>
```

Comment:
Seems this is the only unintended change

Reply:
Think it's worth taking out of the PR?

Reply:
I'll just remove it via meld and -f push